collect_app/src/main/java/org/odk/collect/android/utilities/ExternalWebPageHelper.java

```diff
@@ -59,16 +62,23 @@ public CustomTabsServiceConnection getServiceConnection() {
         return serviceConnection;
     }
 
-    public void openUri(Context context, Uri uri) {
+    public void openWebPageInCustomTab(Activity activity, Uri uri) {
```

Comment:
I didn't actually want this behaviour for the new help button. I'm also thinking we should stop using it for the website and forum links as it can prevent the user from multitasking between Collect and the web page (which they may need to do to read docs or post a problem on the forum). The custom tabs behaviour could useful for the `URLWidget` but honestly it still feels like a lot of code to maintain given the value it brings. Additionally, I just dislike the idea of using a special feature of the Chrome browser (that other browsers are then forced to implement) but that's just me being grouchy.

Interested in thoughts from @lognaturel and @grzesiek2010. It looks like the behaviour was introduced in https://github.com/getodk/collect/issues/563 (thanks for digging that up @grzesiek2010) and the docs on the feature are [here](https://developers.google.com/web/android/custom-tabs).

Reply:
I don't have a strong opinion and would rather let this go for now.